tests/integration/goldens/credentials/google/iam/credentials_v1/services/iam_credentials/async_client.py

```diff
@@ -23,6 +23,7 @@
 from google.api_core import exceptions as core_exceptions
 from google.api_core import gapic_v1
 from google.api_core import retry_async as retries
+import google.auth
```

Comment:
Why do we need `auth` here and in other files? I don't see references to `auth.*` in the file.

Reply:
addressed.